gravityflow/gravityflowformconnector
# Fix field values being filtered during entry update outside of the form submission step

re: [HS#14070](https://secure.helpscout.net/conversation/1217702230/14070/)

Adding the `gform_save_field_value` filter has been moved to the end of `filter_gform_validation` so the filter is only added during a valid submission. It also adds the `gform_entry_created` hook which is used to remove the `gform_save_field_value` filter after the entry has been created from the submission.

This fixes an issue where the `gform_save_field_value` filter is running when other steps processed during the same submission use `GFAPI::update_entry()`.

The extension steps have also been updated to write to the extension log instead of the Gravity Flow log.

## Testing Instructions
- See: HS#14070](https://secure.helpscout.net/conversation/1217702230/14070/#thread-3487686837)
- With this branch repeat steps and find that the values are empty when the entry is viewed on the user input step
